contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
 
 Implements the IBM Security Advisor pipeline
 
+The security advisor filters the flow data obtained from skydive, performs a data transformation, and saves the information to an object store in JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+We use the term "pipeline" to describe the sequence of operations performed on the data: classify, filter, encode, store.
+In the future, we plan to have numerous pipelines performing different kinds of operations.
 
-## Setup Skydive
+## Deploy Skydive
 
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
+
+### All-in-one version
 
 ```
 make
 cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
+sudo $(which skydive) allineone -c skydive.yml
+```
+
+### Multi-node Deployment
+Alternatively, run a skydive agent on each of your hosts and a skydive analyzer on one host.
+
+On each machine:
+
+```
+sudo skydive agent -c skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydive.yml to point to the analyzer.
+
+On one machine:
+
+```
+sudo skydive analyzer -c skydive.yml
+```
+
+
+## Create Security Advisor (Pipeline) Configuration File
+
+Start from the secadvisor.yml.default in the secadvisor directory.
+
+
+```
+cp secadvisor.yml.default secadvisor.yml
+```
+
+The secadvisor.yml is used as a parameter when running secadvisor from the command line.
+
+The default secadvisor.yml has the following fields:
+
 ```
+host_id: ""
+analyzers:
+  127.0.0.1:8082
+pipeline:
+  analyzer:
+    subscriber_url: ws://127.0.0.1:8082/ws/subscriber/flow
+    subscriber_username:
+    subscriber_password:
+  classify:
+    # cluster_net_masks:
+      # - 10.0.0.0/8
+      # - 172.16.0.0/12
+      # - 192.168.0.0/16
+  filter:
+    # excluded_tags:
+      # - internal
+      # - other
+      # - ingress
+      # - egress
+  transform:
+    sa:
+      # exclude_started_flows: true
+  store:
+    type: s3
+    s3:
+      # -- client parames --
+      endpoint: http://127.0.0.1:9000
+      region: local
+      bucket: bucket
+      access_key: user
+      secret_key: password
+      # api_key: key
+      # iam_endpoint: https://iam.cloud.ibm.com/identity/token
+      object_prefix: logs
+      # -- bulk store params --
+      max_flows_per_object: 6000
+      max_seconds_per_object: 60
+      max_seconds_per_stream: 86400
+      max_flow_array_size: 100000
+```
+
+Entries that begin with '#' are comments and are ignored. Be sure to include/exclude the appropriate configuration parameters.
+
+Each instance of secadvisor should use a distinct host_id or should use "" (the empty host id).
```

Comment:
we can always use the default of host_id == "", so you can remove this line 99-100